pkgs/development/libraries/libappindicator/default.nix

```diff
@@ -25,7 +24,7 @@ in stdenv.mkDerivation rec {
     sha256 = "17xlqd60v0zllrxp8bgq3k5a1jkj0svkqn8rzllcyjh8k0gpr46m";
   };
 
-  nativeBuildInputs = [ pkgconfig autoconf ];
+  nativeBuildInputs = [ pkgconfig autoconf vala ];
```

Comment:
`gobject-introspection` not needed, or falling back to pre-generated GIRs from the tarball?

Reply:
ah, it's in buildInputs. will move.